### ISSUE TITLE
Xcode 15, OCLint 22.02, oclint: error: compilation contains multiple jobs:

### ISSUE THREAD

**Issue by FirstDKS521** [oclint/oclint#654]
### Issue Summary
A brief but thorough description of the issue.
oclint: error: compilation contains multiple jobs:
### Environment
- Operation system name and version: macOS Sonoma 14.0
- Xcode: 15.0
- OCLint version:22.02
- How OCLint is installed: local build? prebuilt binary downloaded from github? homebrew install? others?
prebuilt binary downloaded from github
### Reproduction Steps
Detailed steps to reproduce the issue.
oclint-json-compilation-database -- -report-type xcode | xcpretty
#### Sample code
oclint-json-compilation-database -- -report-type xcode | xcpretty
```
C
C++
Objective-C
```

Command to run OCLint with the code above.

### Expected Behavior
What do you expect to happen as a result of the reproduction steps?
Generate analysis report
### Actual Behavior
What currently happens as a result of the reproduction steps?
oclint: error: compilation contains multiple jobs:
### Even Better
Is your project open sourced? If yes, can you point us to your repository?
If not, is it possible to make a small project to illustrate the problem?
If not, can you create a gist with your sample code for us?

**Comment by ryuichi-assistant** [comment]
There hasn't been any activity on this issue recently. Due to the high number of incoming GitHub notifications, we have to clean some of the old issues, as many of them have already been resolved with the latest updates.

Please make sure to update to the latest version and check if that solves the issue. Let us know if that works for you by adding a comment.

**Comment by FirstDKS521** [comment]
> There hasn't been any activity on this issue recently. Due to the high number of incoming GitHub notifications, we have to clean some of the old issues, as many of them have already been resolved with the latest updates.
> 
> Please make sure to update to the latest version and check if that solves the issue. Let us know if that works for you by adding a comment.

The latest version of OCLint still has this problem, I hope the author can fix it

**Comment by ARiezn** [comment]
I can confirm that the latest version of OCLint 22.02 still has the problem.

I've got the same error with Xcode 15.2 and macOS Sonoma 14.2.1.

Tried adding `COMPILER_INDEX_STORE_ENABLE=NO` and specifying a single architecture when building:
```
$ xcodebuild COMPILER_INDEX_STORE_ENABLE=NO -configuration Debug -arch x86_64 -sdk iphonesimulator | xcpretty -r json-compilation-database -o compile_commands.json
```
But still running
```
$ oclint-json-compilation-database -- -report-type xcode
```
ends up with `oclint: error: compilation contains multiple jobs:`

Closed issues #616 and #618 seem to relate only to older Xcode 13.

**Comment by yehe01** [comment]
Having the same issue on the same envrionment.

**Comment by shunsongshiyi** [comment]
> Having the same issue on the same environment.

![image](https://github.com/oclint/oclint/assets/68367949/90d9ca6d-04bf-47c8-83c4-658b401cf0f5)
My same issue`oclint: error: compilation contains multiple jobs` fixed by remove `-ivfsstatcache /Users/mrzhao/derivedData/SDKStatCaches.noindex/macosx14.0-23A334-4a76ee93ef69e462914ffe9883de22b8.sdkstatcache` in compile_command.json
then run `oclint-json-compilation-database` will generate oclint report successfully.

**Comment by ryuichi-assistant** [comment]
There hasn't been any activity on this issue recently. Due to the high number of incoming GitHub notifications, we have to clean some of the old issues, as many of them have already been resolved with the latest updates.

Please make sure to update to the latest version and check if that solves the issue. Let us know if that works for you by adding a comment.

**Comment by ryuichi-assistant** [comment]
This issue will be auto-closed because there hasn't been any activity for a few months. Feel free to open a new one if you still experience this problem.

**Comment by Lianghuajian** [comment]
Try the commands below. This setup supports Xcode 15 on ARM64 (not x86_64) Mac, including the fix for '-ivfsstatcache'
```
git clone -b support_xcode15 https://github.com/Lianghuajian/oclint.git oclint 

cd oclint/oclint-scripts && ./make && cd ..
``` @yehe01 @shunsongshiyi
